devlib/utils/types.py

```diff
@@ -68,6 +68,15 @@ def numeric(value):
     """
     if isinstance(value, int):
         return value
+
+    if isinstance(value, basestring):
+        value = value.strip()
+        if value.endswith('%'):
+            try:
+                return float(value.rstrip('%')) / 100
```

Comment:
> Well, you'll get a list of integers which is a not too bad behaviour IMO

The function's contract is to return a numerical value; returning anything else (even a list of numerical values) is bad. The calling code can reasonably expect to plug the output into a mathematical formula, write value into a column in a database, etc.